### ISSUE TITLE
Can't install add-on

### ISSUE THREAD

**Issue by mgntrn** [hassio-addons/addon-homebridge#12]
## Problem/Motivation

Can't install the add-on.

## Expected behavior

When I click 'INSTALL' I expect the add-on to be installed.

## Actual behavior

After a while the connection to the inferface is lost and Home Assistant restarts.

## Steps to reproduce

Click 'INSTALL', wait for about 30 seconds, then the connection is lost, Home Assistant restarts and the add-on is not installed. I can click INSTALL again, then the same thing happens.

Home Assistant 0.52.1

**Comment by frenck** [comment]
Hi @mgntrn,

Sorry to hear you are experiencing problems. The interface of Hass.io is actually inaccurate. It will stop loading on the frontend, while the backend is still processing. 

Please check the supervisor logs for the actual status. I've seen cases up to 30 minutes.

Could you please check and supply the log file in case of an error?

**Comment by mgntrn** [comment]
Hi @frenck,

Thanks for your reply. I'm not sure what went wrong, but HA kept restarting when I tried to install the add-on, without the add-on being installed afterwards. Yesterday I tried again, so I could check the supervisor logs, but: everything worked as expected! I'm not sure what the problem was, because in the meantime there were no new releases of both HA and the add-on.

Anyway, everything is working great now, great work, thanks!

Kaz
